src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.Print.cs

```diff
@@ -25,16 +28,36 @@ internal static unsafe string DerStringToManagedString(byte[] derString)
                 return null;
             }
 
+            using (asn1String)
+            {
+                return Asn1StringToManagedString(
+                    asn1String,
+                    (bio, str, flags) => Asn1StringPrintEx(bio, str, flags));
```

Comment:
Yep, according to the discussion at https://github.com/bartonjs/corefx/commit/f5452f9adfc0fea0f0262075f32e901f56db5771#commitcomment-12070315.